app/views/request/show.html.erb

```diff
@@ -86,6 +86,15 @@
   <% end %>
 
   <p class="subtitle">
+    Robin Houston
+    <span data-tooltip aria-haspopup="true" class="js-tooltip has-tooltip" data-tooltip-content-id="tooltip_id_2nd_example"
+ data-tooltip-text="Valued contributor"
+ data-tooltip-prefix-class="alaveteli"
+ data-tooltip-link="http://www.google.com"
+ >
+ <%= image_tag "contribution-medal.png", :alt => "This user is contributor to the site" %>
```

Comment:
Is the wording correct here? I thought we were showing the medal if they'd played the classification game? This matters in terms of the logic we need for showing the medal.